common/partition/context.go

```diff
@@ -28,8 +28,6 @@ type configKey struct{}
 
 type isolationGroupKey struct{}
 
-const IsolationGroupKey = "isolation-group"
```

Comment:
unclear if this is used or not, if it's actually necessary I'll put it back later